Makefile

```diff
@@ -0,0 +1,28 @@
+local="http://localhost:8080/ipfs/"
+gway="https://ipfs.io/ipfs/"
+domain="refs.ipfs.io"
+record="dmca.denylists.lists"
```

Comment:
That's purely how we organize domains in digitalocean. I made a separate refs.ipfs.io domain there because we might want different TTL or NS settings than for e.g. ipfs.io itself.